tests/test_main.cpp

```diff
@@ -37,6 +37,7 @@
 #include "editor/editor_settings.h"
 #endif // TOOLS_ENABLED
 
+#include "tests/scene\test_sky.h"
```

Comment:
This must be moved down to the `_3D_DISABLED` part

Reply:
3D specific things, `Sky` isn't technically 3D only but it can't be used without it as far as I can tell (all the materials for it are 3D only)